### PR TITLE
[FIX] l10n_mn: Bug of using recordset containing multiple account tags on Command.link().   A traceback occurs if you try to migrate an account with several tags during an upgrade of the module l10n_mn.

### PR DESCRIPTION
**Pull request by cubicsoft**
l10n_mn Migration script working as wrong.

2023-10-18 15:06:32,682 463 CRITICAL db_1155024 odoo.service.server: Failed to initialize database `db_1155024`. 
Traceback (most recent call last):
  File "/home/odoo/src/odoo/16.0/odoo/service/server.py", line 1299, in preload_registries
    registry = Registry.new(dbname, update_module=update_module)
  File "<decorator-gen-16>", line 2, in new
  File "/home/odoo/src/odoo/16.0/odoo/tools/func.py", line 87, in locked
    return func(inst, *args, **kwargs)
  File "/home/odoo/src/odoo/16.0/odoo/modules/registry.py", line 90, in new
    odoo.modules.load_modules(registry, force_demo, status, update_module)
  File "/home/odoo/src/odoo/16.0/odoo/modules/loading.py", line 484, in load_modules
    processed_modules += load_marked_modules(cr, graph,
  File "/home/odoo/src/odoo/16.0/odoo/modules/loading.py", line 372, in load_marked_modules
    loaded, processed = load_module_graph(
  File "/home/odoo/src/odoo/16.0/odoo/modules/loading.py", line 236, in load_module_graph
    migrations.migrate_module(package, 'post')
  File "/home/odoo/src/odoo/16.0/odoo/modules/migration.py", line 189, in migrate_module
    migrate(self.cr, installed_version)
  File "/home/odoo/src/odoo/16.0/addons/l10n_mn/migrations/1.1/post-migrate.py", line 18, in migrate
    account.write({'tag_ids': [Command.link(cashflow_tag.id)]})
  File "/home/odoo/src/odoo/16.0/odoo/fields.py", line 5065, in __get__
    raise ValueError("Expected singleton: %s" % record)
ValueError: Expected singleton: account.account.tag(88, 3)

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_mn/migrations/1.1/post-migrate.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def migrate(cr, version):
             if matching_template:
                 cashflow_tag = matching_template.tag_ids
                 if cashflow_tag:
-                    account.write({'tag_ids': [Command.link(cashflow_tag.id)]})
+                    account.write({'tag_ids': [Command.link(cf.id) for cf in cashflow_tag]})
```
Comment (gawa-odoo) [inline addons/l10n_mn/migrations/1.1/post-migrate.py:18]: I would tend to say that I prefer having it like this
```suggestion
                    account.write({'tag_ids': [Command.link(tag_id) for tag_id in cashflow_tag.ids]})
```

**Comment by gawa-odoo** [comment]
And can you write a commit message and a title that follows the guidelines please? 
You can find them here https://www.odoo.com/documentation/master/contributing/development/git_guidelines.html#commit-message-structure

**Comment by cubicsoft** [comment]
[FIX] l10n_mn: Bug of using recordset contains multiple account tags on Command.link(). This cause is affecting for version upgrade operation.

**Comment by gawa-odoo** [comment]
Hum sorry, but you still didn't change the commit message, you just wrote it in the PR.
(and I would prefer something like this)
```
[FIX] l10n_mn: Bug of using recordset containing multiple account tags on Command.link(). 

A traceback occurs if you try to migrate an account with several tags during an upgrade of the module l10n_mn.```

**Comment by gawa-odoo** [comment]
Fixed in https://github.com/odoo/odoo/pull/142114

**Pull request closed without merging**